### PR TITLE
AUI-2104 Users should not be able to interact to some features of the form builder while it is moving a row or a column

### PR DESCRIPTION
**Pull request by fernandosouza**

### CODE REVIEW

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-form-builder/js/aui-form-builder-field-list.js [hunk 1]
```diff
@@ -304,6 +329,11 @@ A.FormBuilderFieldList  = A.Base.create('form-builder-field-list', A.Base, [], {
             writeOnce: 'initOnly'
         },
```
Comment (thiago-rocha) [inline src/aui-form-builder/js/aui-form-builder-field-list.js]: Missing documentation here.

Comment (fernandosouza) [inline src/aui-form-builder/js/aui-form-builder-field-list.js]: Oh!!! Sorry!

**Comment by fernandosouza** [comment]
@thiago-rocha. I sent the missing documentation. Could you review again, please?

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-form-builder/js/aui-form-builder-field-list.js [hunk 2]
```diff
@@ -305,12 +330,24 @@ A.FormBuilderFieldList  = A.Base.create('form-builder-field-list', A.Base, [], {
...
 
         /**
+         * Attribute responsible for enable or disable add new fields.
+         *
+         * @attribute fields
```
Comment (thiago-rocha) [inline src/aui-form-builder/js/aui-form-builder-field-list.js]: Wrong documentation.

File: src/aui-layout/js/aui-layout-builder-move.js [hunk 3]
```diff
@@ -129,6 +133,17 @@ LayoutBuilderMove.prototype = {
     },
 
     /**
+     * Adds CSS class to indicate that any element is moving.
+     *
+     * @method _clickColMoveTarget
```
Comment (thiago-rocha) [inline src/aui-layout/js/aui-layout-builder-move.js]: Wrong documentation.

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by thiago-rocha** [comment]
Pull request submitted to https://github.com/mairatma/alloy-ui/pull/325. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**